src/recommendationservice/requirements.in

```diff
@@ -1,12 +1,13 @@
 google-api-core==1.23.0
 google-python-cloud-debugger==2.18
+google-cloud-profiler==3.0.7
 grpcio-health-checking==1.33.2
 grpcio==1.33.2
-opencensus==0.7.11
+opencensus==0.8.0
```

Comment:
Question: Why do we manually upgrade `opencensus` as well? Is it because the older version of `opencensus` doesn't work with `rsa==4.7`?

Reply:
I bumped`opencensus` since I saw it was an older lib 
I thought it would be best if we update these while we can (instead of letting them age), but I can bump this back down to `0.7.11`!

Reply:
Awesome. Nope, don't bump it back down.
I just thought it was related to the `rsa` stuff and was confused.
But yes, thanks for updating `opencensus` as well! 👍